apps/wallet-mobile/src/yoroi-wallets/cardano/api/utils.ts

```diff
@@ -61,8 +64,15 @@ export const toPolicyId = (tokenIdentifier: string) => {
   const tokenSubject = toTokenSubject(tokenIdentifier)
   return tokenSubject.slice(0, 56)
 }
-export const toAssetName = (tokenIdentifier: string) => {
-  return hexToUtf8(toAssetNameHex(tokenIdentifier))
+export const toDisplayAssetName = (tokenIdentifier: string) => {
+  return toUntaggedAssetName(tokenIdentifier)
+}
+
+export const toUntaggedAssetName = (tokenIdentifier: string) => {
+  const hexName = toAssetNameHex(tokenIdentifier)
+  const properties = AssetNameUtils.resolveProperties(hexName)
+  const untaggedName = properties.asciiName ?? properties.hexName
+  return untaggedName
 }
```

Comment:
untagged + ascii when possible is the displayed asset name, I think we can collocate.
```suggestion
export const toDisplayAssetName = (tokenIdentifier: string) => {
  const hexName = toAssetNameHex(tokenIdentifier)
  const properties = AssetNameUtils.resolveProperties(hexName)
  const untaggedName = properties.asciiName ?? properties.hexName
  return untaggedName
}

```

Reply:
thanks to review i just notice that we can get the ascii we need to change the ` properties.hexName` to utf8

what do you think?

Reply:
why we can't get the ascii ?

Reply:
I meant, when ascii property is undefined so it fallbacks to hexName